service/src/direct_invocation/ws_direct_server_runner.rs

```diff
@@ -59,6 +59,7 @@ where
 	HF: CreateWsHandler<Handler = H> + Sync + Send + 'static,
 	E: DirectRequest,
 {
+	#[allow(unused)]
 	pub fn new(handler_factory: Arc<HF>, enclave_api: Arc<E>) -> Self {
 		WsDirectServerRunner { handler_factory, enclave_api }
 	}
```

Comment:
all of the 'old' WS server code in the untrusted worker will be removed, issue was created for that: #368